Elementary-Celebrity-Recognition-My-Dear-Watson/SwiftCloudWorkshop2.swift

```diff
@@ -2,6 +2,8 @@
 /// This is the companion code for my conference talk on using server-side Swift
 /// via IBM Cloud Functions hosted on Bluemix.
 ///
+/// https://github.com/DevWithTheHair/Conference-Talks/tree/master/Elementary-Celebrity-Recognition-My-Dear-Watson
```

Comment:
In case people copy and paste the entire file, add a URL here so they can find their way back.

Reply:
That's a great idea!